mylasqlanon.py

```diff
@@ -109,20 +109,20 @@
                 logger.debug (f"No change indicated for {row} {col_name}")
                 continue
             # Faker has no parameters
-            if "faker" in mod_name:
+            if "faker" == mod_name:
```

Comment:
I'd prefer to see these going the other way, e.g. `mod_name == "faker"`.

Reply:
Makes them more human-readable, I think.

Reply:
Yeah, probably, I was probably thinking more Java first here. In Java you put constant first to avoid NPE. I could switch them though. I don't see anything in the anti-patterns or PEP-8 but I can switch them. Speaking of switch too bad there's no switch. ;)

Reply:
Ok latest commit switches these